bouncerscript/docker.d/worker.yml

```diff
@@ -5,7 +5,14 @@ taskcluster_scope_prefix: { "$eval": "TASKCLUSTER_SCOPE_PREFIX" }
 bouncer_config:
   $merge:
     $match:
-      'COT_PRODUCT == "firefox"':
+      'COT_PRODUCT == "firefox" && ENV == "dev"':
```

Comment:
Don't we want to use this same bouncer instance for fake-prod workers?

Reply:
I think it depends on what we want the default to be when pushing to try. I'm not opposed to it, but I seem to recall some concern around using the dev instance by default here.

Reply:
If possible I would like the default to be such that staging releases mostly "just work", i.e.:
- the bounceradmin instance behind release-bouncer-aliases in fake-prod and the bouncer instance behind release-bouncer-check use the same db
- the bouncer instance behind release-bouncer-check redirects to stage ftp, i.e. where fake-prod beetmover uploads artifacts.  AIUI, this means using the new "dev" bouncer.

Does that make sense?

Reply:
Yes, this is exactly what I want too! (Also: it allows us to bouncer Balrog stage at bouncer dev, which means that final verify will be capable of passing.)

I'll update this to point fake-prod at dev bounceradmin.

There will also be a Gecko change needed to update the bouncer instance we use there which I'm currently working on.

Reply:
I've updated this patch to reflect this. We'll also need a change to the relengworker secrets before this deploys.

https://phabricator.services.mozilla.com/D186634 is the Gecko patch we'll want.